### PR TITLE
masurca 3.3.2

### PR DESCRIPTION
**Pull request by jonchang**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/masurca.rb [hunk 1]
```diff
@@ -2,8 +2,8 @@ class Masurca < Formula
   # cite Zimin_2013: "https://doi.org/10.1093/bioinformatics/btt476"
   desc "Maryland Super-Read Celera Assembler"
   homepage "https://masurca.blogspot.com/"
-  url "https://github.com/alekseyzimin/masurca/releases/download/v3.3.1/MaSuRCA-3.3.1.tar.gz"
-  sha256 "587d0ee2c6b9fbd3436ca2a9001e19f251b677757fe5e88e7f94a0664231e020"
+  url "https://github.com/alekseyzimin/masurca/archive/3.3.2.tar.gz"
```
Comment (sjackman) [inline Formula/masurca.rb]:
```suggestion
  url "https://github.com/alekseyzimin/masurca/releases/download/3.3.2/MaSuRCA-3.3.2.tar.gz"
```

**Comment by jonchang** [comment]
Failed due to OOM (despite `ENV.deparallelize`). Restarting the job...

**Comment by sjackman** [comment]
Some build systems don't respect that environment variables that we set with `ENV.deparallelize` and need more coercion.

**Comment by jonchang** [comment]
🎉

**Comment by sjackman** [comment]
Nice work! Thank you, Jon!

**Pull request closed without merging**